build.gradle.kts

```diff
@@ -267,3 +247,19 @@ tasks.register("buildNdkIntegrationTestsArtifacts") {
 }
 
 nightlyTestsCoverageConfig(threshold = 0.9f)
+kover {
+    isDisabled = false
+    disabledProjects = setOf(
+        "instrumented",
+        "sample",
+        "tools",
```

Comment:
Will it work for `tools:unit` and `tools:detekt`? These projects are disabled here, but mentioned in `koverReportAll `task`.

Reply:
Indeed they should be removed from the `koverReportAll` tasks